adding_species_manuscript.tex

```diff
@@ -374,9 +375,9 @@ \section*{Additions to \stdpopsim}
 many of whom expressed interest in adding non-model species to \stdpopsim.
 The hackathon was then structured based on feedback from these participants.
 One month before the hackathon, we organized a final workshop to prepare interested
-participants for the hackathon, by introducing them to  the process of developing
+participants, by introducing them to  the process of developing
 a new species model and adding it to the \stdpopsim code base.
-Roughly 20 scientists participated in the hackathon,
+Roughly 20 scientists participated in the hackathon (and are included as authors on this paper),
```

Comment:
I think they all are?

Reply:
I think that we can keep this statement as is even if not all participants agreed to be authors. I assume that everyone who ended contributing new species or demo models were included as authors.